cql2pgjson/src/main/java/org/folio/cql2pgjson/CQL2PgJSON.java

```diff
@@ -881,15 +882,20 @@ private String queryByLike(String index, DbIndex dbIndex, IndexTextAndJsonValues
       sql = arrayNode(index, node, modifiers, relationModifiers, schemaIndex, vals, targetTable);
     } else {
       String likeOperator = comparator.equals("<>") ? " NOT LIKE " : " LIKE ";
```

Comment:
Either remove the heading and trainling whitespace here, or remove it below when adding lengthCaseComparator.